src/crewai/utilities/events/crewai_event_bus.py

```diff
@@ -70,7 +70,12 @@ def emit(self, source: Any, event: BaseEvent) -> None:
         for event_type, handlers in self._handlers.items():
             if isinstance(event, event_type):
                 for handler in handlers:
-                    handler(source, event)
+                    try:
+                        handler(source, event)
+                    except Exception as e:
+                        print(
```

Comment:
Would we benefit from using `logger.error` from `logging` instead of `print` here?

Reply:
great question! I trying to keep code consistency here. From what I've seen in the codebase, we usually use print to log information for the user instead of logger.*

This is definitely something we should discuss further soon